README.md

```diff
@@ -3,12 +3,8 @@
 [![CI](https://github.com/spryker-eco/computop/actions/workflows/ci.yml/badge.svg)](https://github.com/spryker-eco/computop/actions/workflows/ci.yml)
 [![Latest Stable Version](https://poser.pugx.org/spryker-eco/computop/v/stable.svg)](https://packagist.org/packages/spryker-eco/computop)
 [![License](https://img.shields.io/github/license/spryker-eco/computop.svg?b=master)](https://github.com/spryker-eco/computop)
-
-[![CI](https://scrutinizer-ci.com/g/spryker-eco/computop/badges/build.png?b=master)](https://scrutinizer-ci.com/g/spryker-eco/computop/build-status/master)
 [![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.4-8892BF.svg)](https://php.net/)
 
-Computop module provides main business logic for [Computop](https://computop.com/) payment system integration.
```

Comment:
you should not remove module description

Reply:
Fixed

Reply:
not fixed actually